### PR TITLE
Frontend #155

### PR DESCRIPTION
**Pull request by APfrom3**
All the changes for task number 155 as well as branch 135 and 164(Deepak) have been aggregated into this branch. This includes adding quickviews to database, adding API endpoints to controller and front end functions to fetch, display and remove quickviews.

### CODE REVIEW

File: backend/controllers/user_controller.go [hunk 1]
```diff
@@ -29,10 +29,21 @@ type dataExtract struct {
 
 var (
 	//account_map   = make(map[string]string)
-	client_id     = os.Getenv("CLIENT_ID")
-	client_secret = os.Getenv("CLIENT_SECRET")
+	//client_id     = os.Getenv("CLIENT_ID")
+	//client_secret = os.Getenv("CLIENT_SECRET")
+	client_secret="27c5646e9c70d276c4b487fa9f5d17366f31f2b1"
+	client_id="cfa78f27867826309fb6"
```
Comment (antitux) [inline backend/controllers/user_controller.go]: Secrets should never be hard coded in applications.

File: client/src/Settings.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 let Settings = {
     currentPort: "http://localhost:3000",
-    githubClientId: "d8ca7de576a6e29f75ca",
+    githubClientId: "cfa78f27867826309fb6",
```
Comment (antitux) [inline client/src/Settings.js]: This should be parameterized also.

**Review by antitux — CHANGES_REQUESTED** [review]
We shouldn't be hard coding secrets inside source code. We need to figure out how to get these out of the source code and being read in from environment variables.

File: client/src/Settings.js (hunk 2 above)
Comment (antitux) [reply client/src/Settings.js]: These are something that should probably be read out of the `.env` file for local development. In production these would be set as environment variables at runtime. Also make sure the documentation is updated to add these parameters.

File: backend/controllers/user_controller.go (hunk 1 above)
Comment (antitux) [reply backend/controllers/user_controller.go]: These are something that should probably be read out of the `.env` file for local development. In production these would be set as environment variables at runtime. Also make sure the documentation is updated to add these parameters.

File: client/src/Settings.js (hunk 2 above)
Comment (deepak0797) [reply client/src/Settings.js]: updated the file to read from the env file in new commit so resolving this

**Review by antitux — APPROVED** [review]

**Comment by antitux** [comment]
There are some merge conflicts that need to be resolved before this can be merged.

**Pull request merged**